runner/ansible/roles/1.4.2/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '1.4.2'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    cibadmin -Q --xpath "//primitive[@type='fence_azure_arm']/@type" > /dev/null 2>&1 || exit 0
+    [[ $(crm_attribute -t crm_config -G -n concurrent-fencing --quiet) = "{{ expected[test_id] }}" && exit 0
```

Comment:
Is this working? I miss the closing brackets (`]]`)

Reply:
fixed and really can't imagine how I toasted that one. should have stopped earlier.